lib/sqlalchemy/sql/compiler.py

```diff
@@ -3323,14 +3323,37 @@ def _generate_prefixes(self, stmt, prefixes, **kw):
             clause += " "
         return clause
 
-    def _render_cte_clause(self):
+    def _render_cte_clause(
+        self,
+        nesting_only: bool = False,
```

Comment:
we're still supporting py2.7 for the 1.4 series :(  have to take annotations out